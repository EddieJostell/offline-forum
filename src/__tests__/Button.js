import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Button from '../components/Button';

describe('<Button />', () => {
    it.skip('render the Button', () => {
        render(<Button />);
    });
} )

it.skip('check if HOC Button exists', () => {
    const btn = shallow(<Button />);
    expect(btn.find('[data-test="button"]').exists());
})

it.skip('Has props shown in Snapshot', () => {
    const btn = shallow(<Button onClick="onClick" />);
    expect(btn).toMatchSnapshot();
});


