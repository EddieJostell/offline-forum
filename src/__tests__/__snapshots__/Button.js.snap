// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Has props shown in Snapshot 1`] = `
<button
  className="bg-indigo-dark hover:bg-indigo-darker text-white font-bold py-2 px-4 rounded-full "
  data-test="button"
  onClick="onClick"
/>
`;
